README.md

```diff
@@ -1,48 +1,56 @@
-# Evergreen
+# [Evergreen](https://evergreen.mongodb.com) &middot; [![GitHub license](https://img.shields.io/badge/license-Apache2.0-blue.svg)](https://github.com/evergreen-ci/evergreen/master/LICENSE)
```

Comment:
I've avoided putting our URL in the README out of a concern around web crawlers, because I'm not sure how much load we can really handle. We got hit a while back, and it wasn't great.

I also don't see that it's a convention elsewhere in our code bases to include links to the license or an image in the header of the README. GitHub already links to it.